Libraries/LibLine/Editor.cpp

```diff
@@ -856,8 +863,7 @@ void Editor::handle_read_event()
             }
             reverse_tab = false;
 
-            auto completion_mode = m_times_tab_pressed == 1 ? SuggestionManager::CompletePrefix : m_times_tab_pressed == 2 ? SuggestionManager::ShowSuggestions
-                                                                                                                           : SuggestionManager::CycleSuggestions;
```

Comment:
???
When did this style change happen?

Reply:
Greetings from clang-format-11, it's probably telling you to not nest ternary operators :P

Reply:
Guess it's finally time to delete that clang-format-10-static binary then~